bundles/io/org.openhab.io.dropbox/README.md

```diff
@@ -22,43 +23,21 @@ This service can be configured in the file `services/dropbox.cfg`.
 | uploadfilter | `^([^/]*/){1}[^/]*$,/configurations.*,/logs/.*,/etc/.*` | No | The defaults are specific to openHAB 1.x running on Unix-like systems |
 | downloadfilter | `^([^/]*/){1}[^/]*$,/configurations.*` | No | The defaults are specific to openHAB 1.x running on Unix-like systems |
 
-## Authorize openHAB
 
-You'll have to authorize openHAB to connect to your Dropbox. This is done in a three-step process. openHAB requests a token which is used as a one-time-password to obtain an access token (second step) which will be used for all future requests against Dropbox.
+## Dropbox App
 
-### Step 1: Monitor Log for Authorization Message
+An app must be created on Dropbox in order to get a generated access token.
 
-This service issues a message to the log automatically on first startup. You will find some log entries (also in the console) containing entries like these:
+Follow the steps in [this tutorial](http://www.iperiusbackup.net/en/create-dropbox-app-get-authentication-token/)
```

Comment:
Could you paraphrase that tutorial here, so this doc is free-standing and doesn't rely on the external link?  If your paraphrase is too close for comfort, please feel free to cite the source of your inspiration.